ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -832,14 +839,19 @@ disambiguateSymbol pm Diagnostic {..} (T.unpack -> symbol) = \case
                     else hideSymbol symbol <$> imps
                 | ImplicitPrelude imps <- hiddens0
                 ]
-    (ToQualified qualMod) ->
+    (ToQualified parensed qualMod) ->
         let occSym = mkVarOcc symbol
             rdr = Qual qualMod occSym
-         in [ Rewrite (rangeToSrcSpan "<dummy>" _range) $ \df -> do
-                liftParseAST @(HsExpr GhcPs) df $
+            asExpr = parensed && isSymOcc occSym
```

Comment:
Can we remove `isSymOcc` here?

Reply:
I've just changed it to use `parensed` only. Thanks!